Ktisis/Common/Extensions/GameObjectEx.cs

```diff
@@ -11,10 +11,30 @@
 using FFXIVClientStructs.FFXIV.Client.Graphics.Scene;
 using CSGameObject = FFXIVClientStructs.FFXIV.Client.Game.Object.GameObject;
 
+using Ktisis.Editor.Context.Types;
+using Ktisis.Actions.Attributes;
+using Ktisis.Core;
+using Ktisis.Core.Types;
+using Ktisis.Core.Attributes;
+using Ktisis.Actions.Types;
+
 namespace Ktisis.Common.Extensions;
 
 public static class GameObjectEx {
-	public static string GetNameOrFallback(this IGameObject gameObject) {
+	public unsafe static bool IsPcCharacter(this IGameObject gameObject) {
+		var csPtr = (CSGameObject*)gameObject.Address;
+		return csPtr != null && csPtr->GetObjectKind() == ObjectKind.Pc;
+	}
+
+	public unsafe static string GetNameOrFallback(this IGameObject gameObject, bool incognito = false) {
```

Comment:
How hard would it be to allow this to be a nullable/enum value and then the absence of the incognito param would default to config?

ex:
```cs
IGameObject char = ...;

var name = char.GetNameOrFallback(); //defaults to config
var anon = char.GetNameOrFallback(true); //Forces the "Actor ..."
var real = char.GetNameOrFallback(false); //gets their actual name
```